float32 motor1_value
float32 motor2_value